### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by joshhornby** [iStefo/ember-select-2#89]
I am seeing the following error:

```Path or pattern "bower_components/select2/select2.js" did not match any files```

Steps to repeat: 

1) ```ember install ember-select-2```
2) This gives an error ```The `ember generate` command requires an entity name to be specified. For more details, use `ember help`.``` So I ran ```ember generate ember-select-2``` as this seemed to fix this.
3) If I then start an ember server I get the following error:

```
Path or pattern "bower_components/select2/select2.js" did not match any files
Error: Path or pattern "bower_components/select2/select2.js" did not match any files
    at Object.multiGlob (/Users/joshuahornby/code/marketing-suite/marketing-suite/node_modules/ember-cli/node_modules/broccoli-kitchen-sink-helpers/index.js:202:13)
    at Class.module.exports.CachingWriter.extend.addFiles (/Users/joshuahornby/code/marketing-suite/marketing-suite/node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/concat-with-maps.js:74:13)
    at Class.module.exports.CachingWriter.extend.updateCache (/Users/joshuahornby/code/marketing-suite/marketing-suite/node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/concat-with-maps.js:52:12)
    at /Users/joshuahornby/code/marketing-suite/marketing-suite/node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/broccoli-caching-writer/index.js:92:34
    at lib$rsvp$$internal$$tryCatch (/Users/joshuahornby/code/marketing-suite/marketing-suite/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:489:16)
    at lib$rsvp$$internal$$invokeCallback (/Users/joshuahornby/code/marketing-suite/marketing-suite/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:501:17)
    at lib$rsvp$$internal$$publish (/Users/joshuahornby/code/marketing-suite/marketing-suite/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:472:11)
    at lib$rsvp$asap$$flush (/Users/joshuahornby/code/marketing-suite/marketing-suite/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1290:9)
    at process._tickCallback (node.js:355:11)
```

Looking inside the ```bower_components``` directory I can confirm that the file it is looking for isn't there.

Thanks

**Comment by Keeo** [comment]
This and css file are in dist folder and not in root anymore. There is also another error after installation: ```The `ember generate` command requires an entity name to be specified. For more details, use `ember help`.```

**Comment by mattmcmanus** [comment]
Looks like there is PR to fix this #91

**Comment by iStefo** [comment]
When installing the just released ember-select-2 via npm, this should be fixed now. Thanks @mattmcmanus for pointing out the issue.

Let me know if the problem persists.
